week_1/String/find_all_anagrams/find_all_anagrams.js

```diff
@@ -0,0 +1,67 @@
+// Given two strings long and short, return an array of all the start indices of short's anagrams in long. You may return the answer in any order.
```

Comment:
These comments shouldn't be exercise requirements
```suggestion
// Given two strings long and short, return an array of all the start indices of short's anagrams in long.
```